WordPress/Classes/ViewRelated/Views/List/ListTableViewCell.swift

```diff
@@ -0,0 +1,122 @@
+/// Table view cell for the List component.
+///
+/// This is used in Comments and Notifications as part of the Comments
+/// Unification project.
+///
+class ListTableViewCell: UITableViewCell, NibReusable {
+    // MARK: IBOutlets
+
+    @IBOutlet private weak var indicatorView: UIView!
+    @IBOutlet private weak var avatarView: CircularImageView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var snippetLabel: UILabel!
+    @IBOutlet private weak var indicatorWidthConstraint: NSLayoutConstraint!
+
+    // MARK: Properties
+
+    /// The color of the indicator circle.
+    @objc var indicatorColor: UIColor = .clear
```

Comment:
Should this have a `didSet` to update `indicatorView.backgroundColor` similar to `showsIndicator`?  Without it, the order that `indicatorColor` and `showsIndicator` are called matters.  Call them in the wrong order and the color doesn't update until the next time the cell is refreshed.

Reply:
Good catch, sir. I'll adjust it soon!